tests/test_38_basepeak_all_levels.py

```
@@ -0,0 +1,55 @@
+    assert metrics["BasePeak_MS1_Mean"] == 100.0
+    assert metrics["BasePeak_MS2_Mean"] == 200.0
+
+
+def test_empty_run_returns_nan():
+    exp = oms.MSExperiment()
+    assert math.isnan(max_base_peak_intensity(exp))
+
+
+def test_empty_scans_skipped():
+    exp = oms.MSExperiment()
+    empty = oms.MSSpectrum(); empty.setMSLevel(1); empty.setRT(1.0)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)